package.json

```diff
@@ -86,8 +86,10 @@
     "object-assign": "4.0.1"
   },
   "peerDependencies": {
-    "react": "^15.0.0",
-    "react-dom": "^15.0.0"
+    "react": "^15.5.0",
+    "react-dom": "^15.5.0",
+    "create-react-class": "15.5.0",
```

Comment:
A peer dependency could be a problem here. Does it mean every people need to install the lib to get react-inifinite work ? Most of people use es6 classes.
("^15.5.0" ?)